app/views/layouts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a list of all posts for a user</h1>
```

Comment:
- According to the project requirements, all controllers should have a corresponding view file for each of the controller actions. I.e For the `post#index` action, the corresponding view should be `app/views/posts/index.html.erb`. Please fix this for all controller actions as required 👍 👍